psa/config.js

```diff
@@ -0,0 +1,26 @@
+import { base32 } from 'multiformats/bases/base32'
+
+/** @type {import('../lib.js').Bucket[]} */
+export const buckets = [
+  {
+    name: process.env.S3_DOTSTORAGE_0_BUCKET_NAME,
+    region: process.env.S3_DOTSTORAGE_0_BUCKET_REGION,
```

Comment:
is this just the names we have 0 & 1?

Reply:
Yeah, for historical reasons we had 2 buckets that received some pinning service data. They are in different regions.